src/components/landing/Graphic/styled.ts

```diff
@@ -0,0 +1,16 @@
+'use client';
+
+import styled from 'styled-components';
+import { borderless } from '@styles/common';
+
+interface GraphicAreaProps {
+  height?: '270px';
+}
+
+export const GraphicArea = styled.div<GraphicAreaProps>`
+  ${borderless}
+  padding-top: 32px;
+  background: #d9d9d9;
+  width: 100%;
+  height: ${props => (props.height ? props.height : '280px')};
+`;
```

Comment:
이부분 interface내의 height와 `GraphicArea`내의 height의 default 값이 다른것 같은데.. 의도를 잘 모르겠습니다 🤔
280px을 default로 쓰고 나머지를 정의해서 쓰려고 하신것 같은데.. 그렇다면 아래와 같은 방식이 더 좋아 보입니다

```suggestion
interface GraphicAreaProps {
  height?: string;
}

export const GraphicArea = styled.div<GraphicAreaProps>`
  ${borderless}
  padding-top: 32px;
  background: #d9d9d9;
  width: 100%;
  height: ${props => (props.height ? props.height : '280px')};
`;
```

Reply:
interface 부분 공부가 부족해서 발생한 부분입니다. 짚어 주셔서 감사합니다! 수정하겠습니다.